dr/discovered/ramen-ops/binding.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
If ACM is never to look at the placement and placementdecision, then we don't need the managedClusterSetBinding.